src/AllocationHistory/components/AllocationHistory.js

```diff
@@ -114,7 +114,10 @@ class AllocationHistory extends Component {
 
 AllocationHistory.propTypes = {
   allocationHistory: PropTypes.shape({
-    allocationHistory: PropTypes.object,
+    allocationHistory: PropTypes.shape({
```

Comment:
Only way I could get this to stop moaning on lint.  Not something we've done before so I'm not entirely sure why it's complaining now. `length` and `map`